packages/@vuepress/core/lib/client/components/OutboundLink.vue

```diff
@@ -1,26 +1,30 @@
 <template>
-  <svg
-    class="icon outbound"
-    xmlns="http://www.w3.org/2000/svg"
-    aria-labelledby="outbound-link-title"
-    role="img"
-    focusable="false"
-    x="0px"
-    y="0px"
-    viewBox="0 0 100 100"
-    width="15"
-    height="15"
+  <span
+    :title="openInNewWindowTitle"
+    class="sr-only"
```

Comment:
In this case, I think native text increases the chances of support in combinations of screen readers and browsers.

```html
<span>
  <span class="sr-only">{{ openInNewWindowTitle }}</span>
```

Reply:
Just to be sure I understand you:
```
<span>
  <svg
        class="icon outbound"
        xmlns="http://www.w3.org/2000/svg"
        aria-hidden="true"
        role="img"
        focusable="false"
        x="0px"
        y="0px"
        viewBox="0 0 100 100"
        width="15"
        height="15"
      >
        <path
          fill="currentColor"
          d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z" />
        <polygon fill="currentColor"
          points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9" />
  </svg>
  <span :title="openInNewWindowTitle" class="sr-only"></span>
</span>```

Reply:
As shown in the code snippet above, I incorporated all your comments:
- [X] added a wrapping span for the SVG and sr-span
- [X] moved the parent sr-span to be a sibling (rather then parent of the SVG)
- [X] made the sr-info text as text child of the sr-span instead of title attribute